src/test/java/codeu/controller/UserProfileServletTest.java

```diff
@@ -0,0 +1,65 @@
+package codeu.controller;
+
+
+import codeu.model.data.AboutMeMessage;
+
+
+import codeu.model.data.User;
+import codeu.model.store.basic.AboutMeMessageStore;
+
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.kefirsf.bb.TextProcessor;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+
+
+
+
+
+
+public class UserProfileServletTest {
```

Comment:
Currently this doesn't test anything.  It just does some setup.  If you wanted to get practice doing tests, you can try to add tests for all of your servlet functionality.  If you prefer to get this PR in first, you can skip the tests for now and either come back to them later or move on to the open project.